django/contrib/admin/options.py

```diff
@@ -1678,6 +1678,23 @@ def changelist_view(self, request, extra_context=None):
             'admin/change_list.html'
         ], context)
 
+    def get_deleted_objects(self, objs, opts, request, using):
+        """
+        Find all objects related to ``objs`` that should also be deleted.
+        ``objs`` must be a homogenous iterable of objects (e.g. a QuerySet).
+
+        Returns a nested list of strings suitable for display in the
+        template with the ``unordered_list`` filter.
+
+        By default this just returns
```

Comment:
Maybe `By default, return the django.contrib.admin.utils.get_deleted_objects.` instead of `By default this just returns django.contrib.admin.utils.get_deleted_objects.`.